src/scripts/seeder.ts

```diff
@@ -176,35 +215,35 @@ async function isAdminUserPresent() {
 	return admins > 0;
 }
 
-async function addAdmin(email: string, password: string, organizationIdentifier: string | null) {
-	await addUserWithPermission(email, password, PermissionFlag.ADMIN_PERMISSION, organizationIdentifier);
+async function addAdmin(email: string, password: string, boroughOrganizationIdentifiers: string[]) {
+	await addUserWithPermission(email, password, PermissionFlag.ADMIN_PERMISSION, boroughOrganizationIdentifiers);
 }
 
-async function addTSBOrganization() {
+async function addBorough(borough: Borough) {
```

Comment:
Kleinigkeit: Hier würde ich ruhig "Organization" im Funktionsnamen drin lassen:

```suggestion
async function addBoroughOrganization(borough: Borough) {
```